File: src/org/koreader/device/rockchip/RK3368EPDController.java
/** RK3368, tested on a Boyue Likebook Mimas */

package org.koreader.device.rockchip;

import android.view.View;

import org.koreader.device.EPDController;

/** we don't care about view on this driver */
@SuppressWarnings("unused")
public class RK3368EPDController extends RK33xxEPDController implements EPDController {
    @Override
    public void setEpdMode(View targetView, String epdMode) {
        requestEpdMode(epdMode);
    }
}

File: app/src/org/koreader/launcher/device/epd/rockchip/RK3368EPDController.kt
/* generic EPD Controller for Android devices,
 * based on https://github.com/unwmun/refreshU */

package org.koreader.launcher.device.epd.rockchip

import org.koreader.launcher.interfaces.EPDInterface

class RK3368EPDController : RK33xxEPDController(), EPDInterface {
    override fun resume() {}
    override fun pause() {}
    override fun setEpdMode(targetView: android.view.View,
                            mode: Int, delay: Long,
                            x: Int, y: Int, width: Int, height: Int, epdMode: String?)
    {
        requestEpdMode(epdMode!!)
    }
}
